storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -53,5 +53,30 @@ include::modules/persistent-storage-csi-vsphere-top-aware.adoc[leveloffset=+1]
 .Additional resources
 * https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-esxi-management/GUID-16422FF7-235B-4A44-92E2-532F6AED0923.html?hWord=N4IghgNiBcIC5gOYgL5A[VMware vSphere tag documenation]
 
+== vSphere VM and persistent disks encryption
+
+You can encrypt virtual machines (VMs) and dynamically provisioned persistent volumes (PVs) on {product-title} running on top of vSphere. 
```

Comment:
We should say - before you can encrypt persistent volumes created by CSI driver, you must encrypt the VMs being used in Openshift cluster and then link to UPI and other docs.

Reply:
Done.